clang/docs/ReleaseNotes.rst

```diff
@@ -74,6 +74,22 @@ C++ Specific Potentially Breaking Changes
     template <typename T>
     void f();
 
+- During constant evaluation, comparisons between different evaluations of the
+  same string literal are now correctly treated as non-constant, and comparisons
+  between string literals that cannot possibly overlap in memory are now treated
+  as constant.
```

Comment:
Could you mention the CWG issue here as well?

Reply:
Done.